great_expectations/cli/v012/datasource.py

```diff
@@ -1243,7 +1246,7 @@ def _get_batch_kwargs_from_generator_or_from_file_path(
                         "\nDoes this file contain a header row?", default=True
                     )
                     batch_kwargs["reader_options"] = {"header": header_row}
-                batch = datasource.get_batch(batch_kwargs=batch_kwargs)
+                datasource.get_batch(batch_kwargs=batch_kwargs)
```

Comment:
@Kilo59 How about `_ = ...` to show that there is a return value, we are just not using it.  Thanks.

Reply:
Done